deploy/kustomize/base/ds-rezolus.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: rezolus
+spec:
+  selector:
+    matchLabels: {}
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "4242"
+        prometheus.io/path: "/metrics"
+    spec:
+      hostNetwork: true
+      hostPID: true
+      containers:
+        - name: rezolus-agent
+          image: rezolus
+          args:
+            - /config/agent.toml
+          securityContext:
+            privileged: true
+            capabilities:
+              add:
+                - SYS_ADMIN
+                - SYS_PTRACE
+                - NET_ADMIN
+                - SYS_RESOURCE
+                - IPC_LOCK
+                - NET_RAW
+          volumeMounts:
```

Comment:
Are all of these necessary?

Reply:
Yes. These are needed otherwise the rezolus was not giving some of the metrics.

Reply:
I'm surprised it needs NET_RAW.

Reply:
I'm surprised, I don't think we open /dev for anything.